src/main/java/org/terasology/tutorialpathfinding/systems/SpawnSystem.java

```diff
@@ -84,10 +81,10 @@ public void blockchanged(OnChangedBlock event, EntityRef entityRef) {
 
         if (spawner.equals(newBlock)) {
             logger.error("spawner placed");
-            spawnerPositions.add(JomlUtil.from(event.getBlockPosition()));
+            spawnerPositions.add(event.getBlockPosition());
```

Comment:
create a new copy of the block position (I don't think we create a defensive copy in the getter).